### PR TITLE
#66 revised adapter_response handler

### PR DESCRIPTION
**Pull request by poludnev**

### CODE REVIEW

File: src/helpers/handlers.ts [hunk 1]
```diff
@@ -84,113 +80,119 @@ const getScreenshotWithPuppeteer = async (
...
-  const jobs = await uploadQueue.getJobs(['active']);
+  try {
+    const { tweetId, userId } = request.query as {
+      tweetId: string;
+      userId: string;
+    };
+    const jobs = await uploadQueue.getJobs(['active']);
 
-  const activeJob = jobs.find((job) => job.data.userId === userId);
+    const activeJob = jobs.find((job) => job.data.userId === userId);
```
Comment (utgarda) [inline src/helpers/handlers.ts:90]: Try `uploadQueue.getJobFromId`
https://github.com/OptimalBits/bull/blob/bae6427ce9d9fac26b198402068bd84647fd8208/lib/queue.js#L240

**Review by utgarda — APPROVED** [review]
LGTM
please take care of minor things mentioned in the review and merge

**Review by utgarda — APPROVED** [review]
LGTM
please take care of minor things mentioned in the review and merge

File: src/helpers/handlers.ts (hunk 1 above)
Comment (poludnev) [reply src/helpers/handlers.ts:90]: our current bulls ver doesn't have getJobFromId method, or it is unaccessible tor some reasons.

image

Anyway, in the line 90 I'm not searching the job by id, as we don't have any maps for userId => jobId, I'm looking for a job that contains certain userId in its data, so getJobFromId is not helpful.

**Pull request merged**